xbmc/music/infoscanner/MusicInfoScanner.cpp

```diff
@@ -42,7 +42,6 @@
 #include "settings/Settings.h"
 #include "FileItem.h"
 #include "guilib/LocalizeStrings.h"
-#include "utils/StringUtils.h"
```

Comment:
IMO you shouldn't rely on headers being included by other headers. If you explicitly use `StringUtils` then also include `utils/StringUtils.h`. That way someone else can remove all usage of `CUtil` and also the `Util.h` header without any additional problems.